india_forecast_app/app.py

```diff
@@ -238,6 +241,50 @@ def save_forecast(
             ml_model_version=ml_model_version,
         )
 
+    if use_adjuster:
+        log.info(f"Adjusting forecast for site_id={forecast_meta['site_uuid']}...")
+        me_values = get_me_values(
+            db_session, forecast_meta["timestamp_utc"].hour, site_uuid=forecast_meta["site_uuid"]
+        )
+
+        log.info(f"ME values: {me_values}")
+
+        # clip me values by 10% of the capacity
```

Comment:
Good idea to add this safeguard in for now, would be interesting to perhaps log  if the min and max were outside this range would perhaps help if you want to change this value in the future

Reply:
good idea